### ISSUE TITLE
Side bearings

### ISSUE THREAD

**Issue by skalyan91** [Tosche/Alcarin-Tengwar#7]
![Screen Shot 2022-01-15 at 9 52 31 pm](https://user-images.githubusercontent.com/2990272/149620817-0d10884c-58ca-4fcb-9d93-f6feaded9dee.png)

The fourth set of letters above looks reasonably well-spaced, but the other three look inconsistent—the spaces between adjacent stems are greater than between adjacent bowls, or between a stem and a bowl.

I think in some cases the side bearings of stems need to be reduced, and in other cases the side bearings of bowls need to be increased. Note that this is not a matter of hinting, as the issue persists at large sizes.

**Comment by Tosche** [comment]
I do not think this is an issue. Stems having larger side bearings are the basics of spacing. I agree some pairs could be kerned slightly though (maybe in the future, but no promises).

**Comment by skalyan91** [comment]
The issue was not with stems having larger side bearings than bowls, but rather with some stems having larger side bearings than other (equivalent) stems in other letters. To me, the space between the stems of _calma_ and _tinco_ looks greater than the spacing between the stems of _anga_ and _ando_; and both of these look greater than the spacing between the stems of _anca_ and _anto_.

It’s still possible that I’m looking at the letters at the wrong size, though….
